data.sql

```diff
@@ -33,3 +33,58 @@ update animals SET owner_id = 3 WHERE (name = 'Devimon' OR name = 'Plantmon');
 update animals SET owner_id = 4 WHERE (name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom'); 
 update animals SET owner_id = 5 WHERE (name = 'Angemon' OR name = 'Boarmon'); 
 
+--PROJECT 4
+INSERT INTO vets (fullname,age, date_of_graduation) VALUES ('William Tatcher', 45 , '2000-04-23');
+INSERT INTO vets (fullname,age, date_of_graduation) VALUES ('Maisy Smith', 26 , '2019-01-17');
+INSERT INTO vets (fullname,age, date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets (fullname,age, date_of_graduation) VALUES ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (species_id, vets_id) VALUES ((SELECT id FROM species WHERE name='Pokemon'),(SELECT id FROM vets WHERE name ='William Tatcher'));
+INSERT INTO specializations (species_id, vets_id) VALUES ((SELECT id FROM species WHERE name='Digimon'),(SELECT id FROM vets WHERE name ='Stephanie Mendez'));
+INSERT INTO specializations (species_id, vets_id) VALUES ((SELECT id FROM species WHERE name='Pokemon'),(SELECT id FROM vets WHERE name ='Stephanie Mendez'));
+INSERT INTO specializations (species_id, vets_id) VALUES ((SELECT id FROM species WHERE name='Digimon'),(SELECT id FROM vets WHERE name ='Jack Harkness'));
+
+
```

Comment:
![image](https://github.com/danutibe07/vet_clinic_database/assets/93039359/e4072175-e01b-45a1-8ad8-ae11dc46dc0f)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍